txpool/txpool.go

```diff
@@ -956,7 +1015,7 @@ func (a *localAccounts) addAddr(addr types.Address) {
 	a.accounts[addr] = true
 }
 
-// numSlots calculates the number of slots for given transaction
-func numSlots(tx *types.Transaction) uint64 {
+// slots() calculates the number of slots for given transaction
+func slots(tx *types.Transaction) uint64 {
```

Comment:
Maybe rename to slotsRequired to improve readability.

Reply:
An alternative I had in mind is `slotWeight()`.

Reply:
+1 on `slotsRequired`

Reply:
[Renamed](https://github.com/0xPolygon/polygon-sdk/pull/213/commits/1cf77747ebc905fe88b13247fcd0357562973f9c)